src/quo/components/tags/collectible_tag/view.cljs

```diff
@@ -1,69 +1,71 @@
 (ns quo.components.tags.collectible-tag.view
   (:require
+    [oops.core :as oops]
     [quo.components.icon :as icons]
     [quo.components.markdown.text :as text]
     [quo.components.tags.collectible-tag.style :as style]
     [quo.theme]
     [react-native.core :as rn]
     [react-native.hole-view :as hole-view]
+    [reagent.core :as reagent]
     [schema.core :as schema]))
 
 (def ?schema
   [:=>
    [:catn
     [:props
      [:map {:closed true}
-      [:options {:optional true} [:maybe [:enum false :add :hold]]]
-      [:size {:optional true} [:enum :size-24 :size-32]]
-      [:blur? {:optional true} :boolean]
+      [:options {:optional true} [:maybe [:enum :add :hold]]]
+      [:size {:optional true} [:maybe [:enum :size-24 :size-32]]]
+      [:blur? {:optional true} [:maybe boolean?]]
```

Comment:
Why `boolean?` over `:boolean` ?

Reply:
There's no guideline for this, but as far as I remember using predicate functions should be quicker, but nothing noticeable. Generally I have no preference of using one over the other